lib/cartopy/tests/mpl/test_mpl_integration.py

```diff
@@ -130,6 +130,37 @@ def test_global_scatter_wrap_no_transform():
     plt.scatter(x, y, c=data)
 
 
+@pytest.mark.natural_earth
+@ImageTesting(['global_hexbin_wrap'],
+              tolerance=12.4 if MPL_VERSION < '2.1.0' else 0.5)
+def test_global_hexbin_wrap_new_transform():
+    ax = plt.axes(projection=ccrs.PlateCarree())
+    # By default the coastline feature will be drawn after patches.
+    # By setting zorder we can ensure our hexbin polygons are drawn
+    # after the coastlines.
+    ax.coastlines(zorder=0)
+    x, y = np.meshgrid(np.linspace(-180, 180, 200), np.linspace(-90, 90, 200))
```

Comment:
The problem is that now you aren't testing the "wrapping" capabilities here.

Reply:
I dropped the transform test as everything is being transformed now.